Mage.Sets/src/mage/sets/Mirage.java

```diff
@@ -43,6 +43,7 @@ private Mirage() {
         cards.add(new SetCardInfo("Bad River", 324, Rarity.UNCOMMON, mage.cards.b.BadRiver.class));
         cards.add(new SetCardInfo("Barbed Foliage", 207, Rarity.UNCOMMON, mage.cards.b.BarbedFoliage.class));
         cards.add(new SetCardInfo("Barbed-Back Wurm", 105, Rarity.UNCOMMON, mage.cards.b.BarbedBackWurm.class));
+        cards.add(new SetCardInfo("Basalt Golem", 259, Rarity.UNCOMMON, mage.cards.b.BasaltGolem.class));
```

Comment:
According to Scryfall, Basalt Golem is number 294 in Mirage, not number 259. This is giving me the image for Circle of Despair (which isn't implemented either, so apparently not caught by the verify test)

Reply:
Odd, gen-card.pl gave me 259. I've noticed that happen for some Alliances cards too, what's up with that?

Reply:
@xenohedron verify test has cards checking, but it disabled by default due to too much errors in outdated sets and cards, see `test_checkWrongCardsDataInSets`. Same for tokens checks: disabled by default in `test_checkMissingTokenData`.

@lagdotcom cause py-scripts uses `mtg-cards-data.txt` -- that file updates manually, so can be very outdated. Use scryfall for actual data.